README.md

```diff
@@ -1,3 +1,11 @@
+# Frontera
+
+## Overview
+
+Frontera is a framework for building [crawl frontiers](http://nlp.stanford.edu/IR-book/html/htmledition/the-url-frontier-1.html). Designed with Scrapy in mind, but applicable to any web crawling project.
```

Comment:
I think "framework implementation" is OK, because it's a framework based implementation of a crawl frontier as opposed to a framework that just happens to implement a crawl frontier.